index.bs

```diff
@@ -11290,6 +11290,35 @@ defined below, skipping over those channels not present.
 		<tr><td>17 <td>SPEAKER_TOP_BACK_RIGHT <td> <td> <td> <td>
 </table>
 
+<h3 id="channels-tail-time">
+Implication of tail-time on input and output channel count</h3>
+
+When an {{AudioNode}} has a non-null <a>tail-time</a>, and an output channel
+count that depends on the input channels count, any change in input channel
+count MUST take into account the {{AudioNode}}'s <a>tail-time</a>.
+
+When there is a decrease in input channel count, the change in output channel
+count MUST happen after the <a>tail-time</a> has passed.
+
+When there is an increase in input channel count, the behavious depends on the
+{{AudioNode}} type:
+
+- For a {{ConvolverNode}}, since the number of input channel determines the
+	<a href="#Convolution-channel-configurations"> audio processing algorithm </a>
+	to use, it MUST immediately start outputing two channels.
+
+	Note: this only applies to the case where the {{ConvolverNode}}'s impulse
+	response is mono. Otherwise, the {{ConvolverNode}} always outputs a stereo
+	signal regardless of its input channel count.
+
+- For other {{AudioNode}}s that have a <a>tail-time</a>, the number of output
+	channels MUST increase after the <a>tail-time</a> has passed.
+
+Note: Intuitively, this allows not losing stereo information as part of
```

Comment:
I am not sure if this section is normative or not. If so, we can block this paragraph as an informative note.

Reply:
Notes are informative in specs. I don't think this is necessary to implement the correct behaviour, but I like the way @karlt explained it, I felt like it could make understanding the issue easier.

Reply:
I was wondering if `Note:` is a magic keyword that puts this paragraph into a green box in the rendered document.

Reply:
Yes, it's a shorthand to make a div that is an informative note.